rapid7_insightidr/komand_rapid7_insightidr/actions/assign_user_to_investigation/action.py

```diff
@@ -0,0 +1,42 @@
+import komand
+from .schema import AssignUserToInvestigationInput, AssignUserToInvestigationOutput, Input, Output, Component
+# Custom imports below
+import json
+from komand_rapid7_insightidr.util.endpoints import Investigations
+from komand.exceptions import PluginException
+
+class AssignUserToInvestigation(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='assign_user_to_investigation',
+                description=Component.DESCRIPTION,
+                input=AssignUserToInvestigationInput(),
+                output=AssignUserToInvestigationOutput())
+
+    def run(self, params={}):
+        investigation_id = params.get(Input.ID)
+        user_email = params.get(Input.USER_EMAIL_ADDRESS)
+
+        endpoint = Investigations.set_user_for_investigation(self.connection.url, investigation_id)
+        payload = {
+            "user_email_address": user_email
+        }
+
+        response = self.connection.session.put(endpoint, json=payload)
+        try:
+            response.raise_for_status()
+        except Exception as e:
+            raise PluginException(cause="The IDR API returned an error",
```

Comment:
Plugin exceptions missing punctuation
```suggestion
            raise PluginException(cause="The IDR API returned an error.",
```